lib/objects/objectsInMemFileDB.js

```diff
@@ -98,6 +101,27 @@ class ObjectsInMemoryFileDB extends InMemoryFileDB {
         }
     }
 
+    /**
+     * Checks if given Id is a meta object, else throws error
+     *
+     * @param {string} id to check
+     * @throws Error if id is invalid
+     */
+    async validateMetaObject(id) {
+        if (this.metaCache[id] === undefined) {
+            // if not cached -> getObject
+            const obj = await this.getObjectAsync(id);
+            if (obj && obj.type === 'meta') {
+                this.metaCache[id] = true;
```

Comment:
I miss a "resolve" here ... or ?!

Reply:
aahhh or is the promise only there for the "exception"?

Reply:
It's basically not needed, code is resolved with last line of code being successfully executed. It would also be the same to use `return` as `return Promise.resolve()`. And as we know return after the last line of code is unnecessary. But I am also not 100 % sure what is best practice here.

Reply:
@AlCalzone what do you think here? (My question was just becasue of "not knowing" :-)

Reply:
Async methods implicitly return Promises with the value after `return`. They also implicitly return rejected Promises if you throw an Error.

Reply:
ok, so we can leave as it is  great!